README.Rmd

```diff
@@ -79,3 +79,412 @@ results(g)[1:5,]
 ## License
 
 MIT © 2024 rOpenSci
+
+## Contributors
```

Comment:
should this be in the contributing guide so anyone knows how to update?

Reply:
Yes, but this package doesn't actually have a contributing guide. Shall be addressed via https://github.com/ropensci-review-tools/goodpractice/issues/185. Thanks